### ISSUE TITLE
Track volume slider: change tooltip from "gain" to "volume"

### ISSUE THREAD

**Issue by saintmatthieu** [audacity/audacity#7097]
The "gain" slider of tracks controls amplification of the waveform _after_ the track's effect stack. In that regard, it actually is a "volume" slider.

**Comment by petersampsonaudacity** [comment]
@saintmatthieu 

Note that the tooltip has "Gain" capitalized - (as does the "Pan" slider)

Accordingly I modififed the issue title with capita;ization.

_Do I assume correctly that this is not for 3.6.2, but for a later release?_

**Comment by saintmatthieu** [comment]
Yes, I don't think there's a big rush, so that'd be 3.7.

**Comment by saintmatthieu** [comment]
@petersampsonaudacity I reckon the changes should be reflected in the documentation ; will you take care of this or should I ?

**Comment by petersampsonaudacity** [comment]
@saintmatthieu 

I wa planning to handle that - in the approprire places I was planning to replace "gain" with "gain/volume" so it becomes agnostic in this regard.

_Partly because there are many places in the manual where gain is used where it is entirely appropriate )in some of the Effects for example)._

**Comment by saintmatthieu** [comment]
Absolutely. The QA list tells what to test ; I guess only the places of the manual that refer to these must be updated, the rest not. Like you said, in other places, "gain" is indeed "gain".

**Comment by chinakov** [comment]
Tested

**Comment by petersampsonaudacity** [comment]
@saintmatthieu 

Hi Matthieu,

Have tou also changed the tooltip for the Gain slider in the Mixer?
![image](https://github.com/user-attachments/assets/a1584837-2742-48f0-8b6a-a2791ac6ee60)

**Comment by saintmatthieu** [comment]
Yes - there should be a QA item for that.

**Comment by petersampsonaudacity** [comment]
Just tested on W11 the Mixer with the latest 3.7.0 alpha.

The tooltip here also says "Volume"

![image](https://github.com/user-attachments/assets/1df73695-fa89-407f-8862-4a5bba81cdfb)

**Comment by petersampsonaudacity** [comment]
I've updated the manual for this.
